client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,98 @@
+import React , {useState, useCallback} from 'react';
+import {Button, Card, Form, Col} from 'react-bootstrap';
+
+export interface BoostCalibrationProps {
+    onSet: (calibValue: number) => void;
+    onReset: () => void;
+    distTravelled: number;
+  }
+
+/**
+ * Boost calibration card, displays the calibrated distance and the 
+ * distance travelled by the bike while also allowing user to set the 
+ * calibrated distance
+ * 
+ * @returns {React.Component} Component
+ */
+export default function BoostCalibration({
+    onSet, onReset, distTravelled
+}: BoostCalibrationProps) {
+    // Controls the feedback form
+    const [validated, setValidation] = useState(false);
```

Comment:
`setValidated` would be a better name, just to keep it similar to the previous variable.